connectors/citrus-testcontainers/src/main/java/org/citrusframework/testcontainers/TestContainersSettings.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.citrusframework.testcontainers;
+
+import java.util.Optional;
+
+public class TestContainersSettings {
```

Comment:
if this is just a utility class, it could be `final`:

```suggestion
public final class TestContainersSettings {
```

I think utility classes are a good use-case for this.